src/components/RegisterPopup/RegisterPopup.test.tsx

```diff
@@ -0,0 +1,54 @@
+const React = require('react');
+const { mount } = require('enzyme');
+const sinon = require('sinon').sandbox.create();
+const chai = require('chai');
+const chaiEnzyme = require('chai-enzyme');
+chai.use(chaiEnzyme());
+const {expect} = chai;
+import RegisterPopup from './RegisterPopup';
```

Comment:
I don't know why use require or put it inside the beforeEach， all the tests will fail